Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+    -   *Filelist*
+
+        The :t3ext:`filelist` is a module in the TYPO3 backend used for
+        displaying and managing files and media resources. It displays contents
+        of all configured file storages. When referencing files from content
+        elements, a popup window will also display the filelist in the backend.
+
+    ..  _flash-message:
+    -   *Flash Message*
+
+        Flash Messages are temporary notifications in the TYPO3 backend that
+        inform users about important events or changes. Also the Extbase
+        framework allows to use an API to display these kinds of messages in the
+        frontend.
+
+    ..  _flexform:
+    -   *FlexForm*
+
+        FlexForms are a selection of functionalities that allow to extend
+        content element settings in the Backend, which can be accessed in the
+        frontend. A flexForm data source (XML format) defines available sheets,
+        sections and fields, which are displayed alongside a record in the
+        backend record editing interface (based on TCA naming).
+        The values entered in a FlexForm data source are also saved as XML data
+        (as some kind of "blob", so it needs serialization and deserialization
+        when being accessed), which allows for any kind of flexible additional
+        data storage in parallel to the relational database tables (like
+        :sql:`tt_content`).
+
+    ..  _fluid:
+    -   *Fluid*
+
+        Fluid is a template engine in TYPO3 used for creating dynamic and
+        customizable frontend layouts. The format looks like HTML and offers
+        specific and extensible tags to be embedded. It also standard variable
+        replacement as well as a large range of algorithmic and logical
+        operations.
+
+    ..  _forge---forger---gerrit:
+    -   *Forge / Forger / Gerrit*
+
+        [Forge](https://forge.typo3.org/) is the central issue platform for
+        letting the Core Team manage the TYPO3 project and its features and
+        bugs. [Forger](https://forger.typo3.com/) and [Gerrit](https://review.typo3.org/)
+        are tools for code review and management.
+
+    ..  _form-framework---form-extension:
+    -   *Form Framework / Form Extension*
+
+        The :t3ext:`form` framework in TYPO3 is used to create and manage
+        complex forms with various fields and validations. A backend module
+        allows to configure these forms with a powerful GUI.
+
+    ..  _form-variants:
+    -   *Form Variants*
+
+        Form Variants are different versions or variations of a form built in
+        the form framework, that can be defined and used within the TYPO3
+        system.
+
+    ..  _formengine:
+    -   *FormEngine*
+
+        The FormEngine is a vital component in TYPO3 responsible for displaying all record
+        and content editing parts in the Backend.
+
+    ..  _fsc---csc:
+    -   *fsc / csc*
+
+        fsc (Fluid Styled Content) and csc (CSS Styled Content) are system
```

Comment:
Do we really still want to explain CSC? IMO it was removed with TYPO3 8 or 9.

Reply:
Yes, IMO to explain TYPO3 syntax and where CSC was a thing it IMO fits to state that here?